demisto_sdk/commands/validate/validators/PB_validators/PB116_is_stopping_on_error.py

```diff
@@ -0,0 +1,51 @@
+from __future__ import annotations
+
+from typing import Iterable, List
+
+from demisto_sdk.commands.content_graph.parsers.related_files import RelatedFileType
+from demisto_sdk.commands.content_graph.objects.playbook import Playbook
+from demisto_sdk.commands.validate.validators.base_validator import (
+    BaseValidator,
+    ValidationResult,
+)
+
+ContentTypes = Playbook
+
+
+def is_indicator_pb(playbook: Playbook):
+    """
+    Check if the playbook has indicators as input query.
+    """
+    return any(
+        (i.get("playbookInputQuery") or {}).get("queryEntity") == "indicators" for i in playbook.data.get("inputs", {})
+    )
+
+
+def get_error_tolerant_tasks(playbook: Playbook):
+    """
+    Retrieve tasks from the playbook that are set to continue on error.
+    """
+    return [task for task in playbook.tasks.values() if task.continueonerror]
+
+
+class IsStoppingOnErrorValidator(BaseValidator[ContentTypes]):
+    error_code = "PB116"
+    description = "The validation checks that all playbook tasks stop when encountering an error."
+    rationale = "If a playbook task does not stop on error, following tasks might rely on its output and fail."
```

Comment:
we should explain why this is this only for indicator pbs. lets also write in the description that its specific to this type

Reply:
done